app/code/Magento/Checkout/Controller/Cart.php

```diff
@@ -132,4 +127,15 @@ protected function getBackUrl($defaultUrl = null)
 
         return $defaultUrl;
     }
+
+    /**
+     * @return bool
+     */
+    protected function shouldRedirectToCart()
```

Comment:
New protected properties/methods are not allowed. Could you change visibility to private? I see it also used in Cart/Add.php controller, so please add it there too